templates/puppet_signscript.rb.erb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+#
```

Comment:
Maybe we should add some small documentation to this to give some context.
`This takes a base64 encoded CSR as input and uploads it to foreman-proxy for analysis. It is used to verify if the CSR should be signed by PuppetCA.`